app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split()
+    if command_split[1] != command_split[2]:
```

Comment:
you can use unpacking here but first check if len(command_split) == 3
After that you need check if command is `cp` and file1 != file2 and than copy file if not do nothing